test/mailers/previews/publisher_mailer_preview.rb

```diff
@@ -85,10 +86,6 @@ def email_user_on_hold
     PublisherMailer.email_user_on_hold(Publisher.first)
   end
 
-  def update_to_tos
```

Comment:
I think if you want to remove this, you should also remove `lib/tasks/email_tos_update.rake` and `update_to_tos` from `config/locales/ja.yml`.

Reply:
Is this used anymore? This was going to be a followup pr, but I'll include it in this one if it's not used.

Reply:
The `update_to_tos` was a one-off script to send a one-off email. We don't send one-off emails from our servers anymore as we just send via SendGrid website instead.

Reply:
I restored it to limit this PR scope. Will issue a followup.